gradle.properties

```diff
@@ -20,7 +20,7 @@ platformDownloadSources = true
 
 # Plugin Dependencies -> https://plugins.jetbrains.com/docs/intellij/plugin-dependencies.html
 # Example: platformPlugins = com.intellij.java, com.jetbrains.php:203.4449.22
-platformPlugins =
+platformPlugins = org.jetbrains.plugins.go
```

Comment:
Do we really need this? I guess that forces everybody to install the go plugin?

I need to test this with say Idea Community.